docs/optional-components/tickstack.md

```diff
@@ -0,0 +1,688 @@
+  * `production.env`  
+
+* You'll edit the Telegraf Agent's empty or default settings to properly monitor the various local ISLE services and send metrics to the local Influxdb database on the Staging system.
+
+- You'll restart your containers with the new services having been added and configured.
+
+* Using a web browser, you'll navigate to the new Chronograf dashboard and complete the configuration of the Influxdb, Chronograf and Kapacitor services. 
+  * You'll have the option to create new dashboards
+  * If desired, you'll add additional alerts
+
+- You'll repeat the steps to edit the Telegraf Agent's configuration for your Production system only. You do not need to install the remaining "ICK" stack on Production though.
+
+* You can then review the `Using Chronograf` section to learn how to review the new added hosts, dashboards and Log Viewer to review ISLE services log streams.
+
+- Lastly and optionally, you can setup alerts in Chronograf to warn you and your team of when certain events or issues have occurred on your newly monitored systems.
+
+---
+
+## Installation
+
+### Assumptions
+
+* Previously installed and running Production and Staging ISLE Host systems are in place already
+
+- You'll need to use the ISLE images tagged as `1.2.0` and higher for the syslog driver changes to be in place.
+
+* That the "sidecar" method will be the installation type running on the Staging system to receive data from Staging and Production.
+
+- A firewall configuration that allows incoming public traffic to port `8086` traffic
+
+* A firewall configuration that allows incoming private traffic access to port `8888`. It is recommended that this not be general public access but whitelisted to a select range of IP addresses given this will be the `Chronograf` dashboard port. Trusted users only please.
+
+- Only installation on Linux based host systems e.g. Debian / Ubuntu 18.04+ and Red Hat / CentOS 7.x+
+  * Attempting to run TICK locally on a MacOS system can fail https://github.com/docker/for-mac/issues/3303
+
+* Other TICK / ISLE configurations are possible but **are not supported or detailed** as of yet in this documentation. Endusers seeking alternative setup possibilities are encouraged to further review and mine from the official Influxdata [documentation](https://docs.influxdata.com/platform/introduction/).
+
+---
+
+### Installation Instructions
+
+* Shut down all running ISLE containers, Staging first, then production second once Staging is up and running again.
+
+---
+
+#### 01. Docker syslog driver setup
+
+* You will need to use the `sudo` command or become the `root` user for the steps below. Copying this file allows the Docker Daemon to use the `syslog` driver for log files and reporting to the `telegraf` agent.
+
+- Check if you already have an existing `/etc/docker/daemon.json` file. 
+  * If **YES**, add / merge  the contents of `/ISLE/config/proxy/tick/docker/daemon.json` with your current `/etc/docker/daemon.json` file.
```

Comment:
this path is incorrect for all the daemon.json paths here: it should be `ISLE/config/tick/docker/daemon.json`